
try:
    pass
except KeyError as err:
    del err

try:
    pass
except KeyError as err:
    err = 5
